site/static/admin/config.yml

```diff
@@ -9,6 +9,7 @@ collections: # A list of collections the CMS should be able to edit
   - name: "post" # Used in routes, ie.: /admin/collections/:slug/edit
     label: "Post" # Used in the UI, ie.: "New Post"
     folder: "site/content/post" # The path to the folder where the documents are stored
+    path: "{{slug}}/index" # Each document is inside a folder with the slug name.
```

Comment:
I didn't test this, but the docs say that the public folder has to be defined per collection if you want to use relative paths.

So I assume that we are missing this:

```
media_folder: ''
public_folder: ''
```

Reply:
I think it does work because it defines those folders by default when the path is set.  "When specifying a path on a folder collection, media_folder defaults to an empty string."  But maybe it's better to explicitly set it?  I don't know.